### ISSUE TITLE
Focus indicators missing in homepage tab buttons

### ISSUE THREAD

**Issue by SaptakS** [freedomofpress/pressfreedomtracker.us#1678]
When we navigate through the tab buttons shown above the data viz, there is no focus indicators.

![image](https://github.com/freedomofpress/pressfreedomtracker.us/assets/9530293/e2fae399-61c0-4100-bb67-f2f4f5e8f61b)

**Comment by harrislapiroff** [comment]
When implementing this, look into extracting these button blocks into a more generalized component for reuse. If it would take too long, just implement focus styles.

**Comment by harrislapiroff** [comment]
One thing I'm thinking about this:

Focus indicators can be kind of annoying and sometimes confusing if you're *not* using keyboard navigation. Is `:focus-visible` reasonable to use these days?

**Comment by SaptakS** [comment]
> Focus indicators can be kind of annoying and sometimes confusing if you're not using keyboard navigation. Is :focus-visible reasonable to use these days?

Yes, `:focus-visible` is what I meant when I said focus indicators.

**Comment by harrislapiroff** [comment]
Let's use `:focus-visible` to add a ~yellow~ thicker black outline to focused buttons on keyboard nav—to match other buttons on site. (Updated to match Saptak's comment below)

**Comment by SaptakS** [comment]
I think we can just treat them like any other button and links and basically give a thicker black outline on `:focus-visible`.

**Comment by harrislapiroff** [comment]
Oh, sure, that sounds correct! Thanks for that @SaptakS
